packages/smart/contracts/turbo/TrustedMarketFactory.sol

```diff
@@ -40,4 +40,8 @@ contract TrustedMarketFactory is AbstractMarketFactoryV3, CalculateLinesToBPoolO
     function getMarketDetails(uint256 _id) public view returns (MarketDetails memory) {
         return marketDetails[_id];
     }
+
+    function getRewardEndTime(uint256 _eventId) public override returns (uint256) {
```

Comment:
TrustedMarketFactory doesn't have an event. Nor does it really have a sensible way to say when a market is expected to end, though you can add that.

TBH I would just make TrustedMarketFactory NOT be a Rewardable.